components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -143,9 +143,27 @@ spec:
           - name: TYK_GW_ENABLENONTRANSACTIONALRATELIMITER
             value: "true"
           - name: TYK_GW_POLICIES_POLICYSOURCE
+        {{- if .Values.global.isDataplane }}
+            value: "rpc"
+        {{- else}}
             value: "file"
+        {{- end}}
+        {{- if .Values.global.isDataplane }}
+          - name: TYK_GW_ANALYTICSCONFIG_TYPE
+            value: "rpc"
+          - name: TYK_GW_SLAVEOPTIONS_USERPC
+            value: "true"
+          - name: TYK_GW_AUTHOVERRIDE_FORCEAUTHPROVIDER
```

Comment:
I can't see this environment variable in the documentation. Can you double check if it is correct?

Reply:
The variable is indeed correct. All config options of tyk are accessible via such env vars that are generated automatically from some json tags in the code. While digging through hybrid tutorials I found these were set at tyk-config.json level and the only way to pass that info to the tyk-gateway was like this.

Reply:
perfect! thanks for the clarification